src/main/java/no/sikt/sws/WorkspaceIndexHandler.java

```diff
@@ -24,11 +27,19 @@ protected IndexResponse processInput(Void input, RequestInfo request, Context co
         var workspace = request.getPathParameter(WORKSPACE_IDENTIFIER);
         var action = request.getPathParameter(RESOURCE_IDENTIFIER);
 
-        var response = httpMethod + " opensearch/" +workspace + "/" + action;
+        var endpoint = OPENSEARCH_ENDPOINT_ADDRESS;
+        var region = AWS_REGION;
+
+        var response = httpMethod + " opensearch/" + workspace + "/" + action
+                + " endpoint: " + endpoint + " region: " + region;
+
+        System.out.println(endpoint);
+
```

Comment:
You dont need to fix this now, but we should within few days start using the logger that is used in NVA instead of a direct System.out.println()